go-tools/bisection-testgen/README.md

```diff
@@ -0,0 +1,80 @@
+# Bisection test fixtures generator
+
+## Usage
+
+### NOTE
+
+Currently, there's some discrepancy between the Go code and protobuf
+declarations. For example, `Validator` struct contains `pub_key` while protobuf
+version uses `pub_key_type` and `pub_key_bytes`. If you look at the code, you
+can notice we're using a json marshaler that converts the `Validator` struct to
+JSON. Ideally, we should first convert all structs to protobuf because Rust
+uses serde and protobuf, and expects the same from Go. So this is kinda a hack
+and something we should fix in the future. Specifically, we should:
+
+1. Use [protojson](https://pkg.go.dev/google.golang.org/protobuf/encoding/protojson) to marshal the structs to JSON.
+2. Double-check Go and Rust protos match.
```

Comment:
3. Go `Validator` and Rust `Validator` should probably use `pub_key_name` and `pub_key_bytes`... not sure